contracts/evoting/mod_test.go

```
@@ -192,6 +192,56 @@ func TestCommand_CastVote(t *testing.T) {
 	dummyElection.Status = types.Open
 	jsElection, _ = json.Marshal(dummyElection)
 
+	_ = snap.Set(dummyElectionIdBuff, jsElection)
```

you should not ignore an error

---

same bellow